content/configuration/passing-a-config.md

```diff
@@ -4,12 +4,30 @@ sort: 2
 contributors:
   - sokra
   - skipjack
+  - kbariotis
 ---
 
-...
+Besides exporting a single config object, there are a few more ways that cover other needs as well.
 
 ?> exporting a function and --env
```

Comment:
Since you're already working on this file, can you replace this TODO with a nice link to https://webpack.js.org/configuration/environment/? :smile: